e2e/features/step_definitions/coreUX.stepdef.ts

```diff
@@ -6,8 +6,10 @@ import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
 
 Given('I am on the strimzi-ui homepage', () => {
   cy.visit('localhost:3000/index.html');
+  cy.injectAxe();
 });
 
 Then(`the welcome message appears`, () => {
   cy.get('#root').contains(`Welcome to the Strimzi UI`);
+  cy.checkA11y();
```

Comment:
https://www.npmjs.com/package/cypress-axe#using-the-violationcallback-argument - I'd like if we had support for this to log out to console.

Otherwise, when test fails - while we can watch the video that GHA uploads as an artifact, it would be good to have failure in the workflow log.

Reply:
Also - Carbon is not always 100% accessible - how do you flag particular rules as "ignore for this element"?

Reply:
Finally - we will need to discuss "what rules should we run" and "what impact level fails". Can you propose a default set?

Reply:
We can disable rules that the frameworks don't support.  On patternfly we try to not disable rules.